lib/core/src/lib/i18n/en.json

```diff
@@ -364,11 +364,6 @@
     "ACTION": {
       "HELP": "NEED HELP?",
       "REGISTER": "REGISTER"
-    },
-    "DIALOG": {
```

Comment:
Please revert those translations as you marked Login component as deprecated. They need to exist until we really remove that component for a case if anyone is still using that